pkg/virt-controller/services/renderresources_test.go

```diff
@@ -54,15 +54,15 @@ var _ = Describe("Resource pod spec renderer", func() {
 	Context("Default CPU configuration", func() {
 		cpu := &v1.CPU{Cores: 5}
 		It("Requests one CPU per core, when CPU allocation ratio is 1", func() {
-			rr = NewResourceRenderer(nil, nil, WithoutDedicatedCPU(cpu, 1))
+			rr = NewResourceRenderer(nil, nil, WithoutDedicatedCPU(cpu, 1, true))
```

Comment:
Why do these test changes?

Reply:
Because `NewResourceRenderer()` changed? I'm not sure what you're asking here...

Reply:
I mean the default should be `false` to preserve the test semantic. Do you agree? We should only add new tests that will use `true`